### PR TITLE
[EAM-2960] Add field 'Assigned by' in EAMlight

### PR DESCRIPTION
**Pull request by purquijo**
Added new field 'Assigned by' in the scheduling of the work order. A new WS method has been created for the autocompletion of the supervisors.

### CODE REVIEW

File: src/ui/pages/work/WorkorderScheduling.js [hunk 1]
```diff
@@ -41,6 +41,12 @@ const WorkorderScheduling = (props) => {
                 autocompleteHandler={WS.autocompleteEmployee}
             />
 
+            <EAMAutocomplete
+                {...register('schedgroup','assignedBy')}
+                barcodeScanner
+                autocompleteHandler={WS.autocompleteSupervisor}  //TODO: We require a new implementation of the autocomplete but for supervisors
```
Comment (ricardo-j-almeida) [inline src/ui/pages/work/WorkorderScheduling.js:47]: While there is no PR for the newly required ws for supervisors, this merge request should be marked as WIP. Once it is ready, it should explicitly reference the backend MR on which it relies.

Comment (purquijo) [reply src/ui/pages/work/WorkorderScheduling.js:47]: Here is the [MR](https://github.com/cern-eam/eam-light-backend/pull/107) for the revisors' autocomplete.

Comment (ricardo-j-almeida) [inline src/ui/pages/work/WorkorderScheduling.js:47]: Is this comments still relevant?

Comment (purquijo) [reply src/ui/pages/work/WorkorderScheduling.js:47]: No

**Pull request merged**